src/Application/Config.php

```diff
@@ -4,14 +4,16 @@
 
 namespace ProfessionalWiki\WikibaseFacetedSearch\Application;
 
+use RuntimeException;
+use Wikibase\DataModel\Entity\ItemId;
 use Wikibase\DataModel\Entity\PropertyId;
 
 class Config {
 
 	public function __construct(
 		public readonly ?string $linkTargetSitelinkSiteId = null,
-		public readonly ?PropertyId $instanceOfId = null,
-		public readonly ?FacetConfigList $facets = null
+		private readonly ?PropertyId $instanceOfId = null,
+		private readonly ?FacetConfigList $facets = null
```

Comment:
I switched to getters, since for access in our application, after the config has been figured out, these values are no longer nullable.